.gitignore

```diff
@@ -40,3 +40,6 @@ config/customization/*
 config/transifex.netrc
 !config/provider/.gitkeep
 !config/customization/.gitkeep
+
+# ignore IDE configuration 
+.idea/
```

Comment:
I bet there are a lot of different ways.
First you might need to know which commit changed it...
Then you can use an interactive rebase and edit the commit in question. Or you could add a new commit and then squash it with the old commit in an interactive rebase.
```bash
git log  -1 .gitignore
git rebase -i
```